Sources/SwiftSyntax/SwiftSyntaxCompatibility.swift

```diff
@@ -186,6 +186,67 @@ public extension FunctionTypeSyntax {
   }
 }
 
+public extension ImportDeclSyntax {
+    @available(*, deprecated, renamed: "unexpectedBetweenModifiersAndImportKeyword")
+    var unexpectedBetweenModifiersAndImportTok: UnexpectedNodesSyntax? {
+        get {
+            return unexpectedBetweenModifiersAndImportKeyword
+        }
+        set(value) {
+            unexpectedBetweenModifiersAndImportKeyword = value
+        }
+    }
+
+    @available(*, deprecated, renamed: "importKeyword")
+    var importTok: TokenSyntax {
+        get {
+            return importKeyword
+        }
+        set(value) {
+            importKeyword = value
+        }
+    }
+
+    @available(*, deprecated, renamed: "unexpectedBetweenImportKeywordAndImportKind")
+    var unexpectedBetweenImportTokAndImportKind: UnexpectedNodesSyntax? {
+        get {
+            return unexpectedBetweenImportKeywordAndImportKind
+        }
+        set(value) {
+            unexpectedBetweenImportKeywordAndImportKind = value
+        }
+    }
+
+    @available(*, deprecated, message: "Use an initializer with importKeyword")
+    init(
+        leadingTrivia: Trivia? = nil,
+        _ unexpectedBeforeAttributes: UnexpectedNodesSyntax? = nil,
+        attributes: AttributeListSyntax? = nil,
+        _ unexpectedBetweenAttributesAndModifiers: UnexpectedNodesSyntax? = nil,
+        modifiers: ModifierListSyntax? = nil,
+        _ unexpectedBetweenModifiersAndImportTok: UnexpectedNodesSyntax? = nil,
+        importTok: TokenSyntax = .keyword(.import),
```

Comment:
The tests don't compile because of `Ambiguous use of 'init'` if we add a default value to `importTok `

For me it's fine to remove default value, as those who use `importTok` is not using the default value.

Reply:
Should we attribute the new deprecated initializer with `@_disfavoredOverload` instead?

Reply:
Added to all initializers